api/inventorypb/agents.proto

```diff
@@ -477,6 +492,7 @@ message AddNodeExporterRequest {
   ];
   // Custom user-assigned labels.
   map<string, string> custom_labels = 2;
+  bool push_metrics = 3;
```

Comment:
comment missed

Reply:
added

Reply:
But it is copied from the wrong place now :) It is copied from `push_metrics_enabled` above (describes state), not from `push_metrics` below (describes action).

Reply:
yep, changed